Adding a using certificate section (SOFTWARE-3999) (#727)
opensciencegrid/docs: docs/security/user-certs.md

        Enter PEM pass phrase:
        Verifying - Enter PEM pass phrase:

> Using your User Certificate

Reviewer: ```suggestion
Using Your User Certificate
```

According to https://titlecase.com/

---------------------------

1. The standard location to place user certificates is in the users certificate home directory in the `.globus` directory:

        :::console
        user@host $ cp mkdir ~/.globus
        user@host $ cp userkey.pem ~/.globus/
        user@host $ cp userhost.pem ~/.globus/
        user@host $ cp usecred.p12 ~/.globus/

1. To generate a proxy use the command `voms-proxy-init`. 

        :::console
      	user@host $ voms-proxy-init

1. (Optional) If user certificates are not in the `.globus` then the path has to be passed to `voms-proxy-init`

        :::console

>         user@host $ voms-proxy-init --cert /pathtto/usercert.pem --key /pathtto/userkey.pem

Reviewer: ```suggestion
        user@host $ voms-proxy-init --cert /<PATH TO>/usercert.pem --key /<PATH TO>/userkey.pem
```

1. In order to find the Distinguised Name (DN), issuer and lifetime of a certificate:

        :::console

>         user@host $ openssl x509 -in /pathtto/usercert.pem -noout -subject -issuer -enddate

Reviewer: ```suggestion
        user@host $ openssl x509 -in /<PATH TO>/usercert.pem -noout -subject -issuer -enddate
```
